client/app/telecom/pack/xdsl/access/profil/pack-xdsl-access-profil.html

```diff
@@ -15,7 +15,7 @@
             data-ng-disabled="access.tasks.current.dslamProfileChange">
         </select>
 
-        <button class="btn btn-primary mt-1"
+        <button class="oui-button oui-button_full-width oui-button_primary mt-3 mb-2"
```

Comment:
-`mt-3 mb-2` + wrap the `<select>` just above with a `<div class="form-group">`.